### PR TITLE
MAT-9427: add a composite bubble for composite measures

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-9427](https://jira.cms.gov/browse/MAT-9427)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/PageHeader/pageHeader.scss [hunk 1]
```diff
@@ -138,6 +138,21 @@
...
+        line-height: 20px;
+        background-color: #f8e1f8;
+        padding: 2px 12px;
+        border: 1px solid #f8e1f8;
+        border-radius: 24px;
+        display: flex;
+        align-items: center;
+        height: 24px;
+        margin-left: 16px;
+      }
```
Comment (adongare) [inline src/components/PageHeader/pageHeader.scss:147]: I would just reuse the draft bubble 
```
> .composite-bubble {
  @extend .draft-bubble;
  background-color: #f8e1f8;
  border-color: #f8e1f8;
}
```
OR move common css to bubble class and then composite-bubble and draft-bubble extending buble clas
```
.bubble {
  // common css
}
> .draft-bubble {
  @extend .bubble;
  background-color: #e1f3f8;
  border-color: #e1f3f8;
}

> .composite-bubble {
  @extend .bubble;
  background-color: #f8e1f8;
  border-color: #f8e1f8;
}
```

**Review by adongare — APPROVED** [review]
